test/unit/protocol/StreamrBinaryMessage.test.js

```diff
@@ -18,15 +21,15 @@ describe('StreamrBinaryMessage', () => {
 
         beforeEach(() => {
             version = 28
-            bytes = new StreamrBinaryMessage(
+            bytes = new StreamrBinaryMessageV28(
                 streamId, streamPartition, timestamp, ttl,
                 StreamrBinaryMessage.CONTENT_TYPE_JSON, Buffer.from(JSON.stringify(msg), 'utf8'),
             ).toBytes()
         })
 
         describe('toBytes/fromBytes', () => {
             it('must not alter the field content', () => {
-                const m = StreamrBinaryMessage.fromBytes(bytes)
+                const m = StreamrBinaryMessageFactory.fromBytes(bytes)
```

Comment:
`StreamrBinaryMessageV28.fromBytes(bytes)` here - let's test `StreamrBinaryMessageFactory` in its own test suite. Similarly elsewhere in this file.